### PR TITLE
feat: Initial setup and add routes 

### PR DESCRIPTION
**Pull request by shubhamkumar648**
#### Files to check
    []agrox-app/src/App.js
    []agrox-app/src/pages/index.js

 PS - Remaining  Ignore other files. They are Work in Progress. 

What not to check
External links

### CODE REVIEW

File: agrox-app/src/App.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    <Routes>
+    <Route path ="/" element ={<Home/>}/>
+    <Route path ="/wishlisht" element ={<Wishlisht/>}/>
+    <Route path ="/product" element ={<Products/>}/>
+    <Route path ="/cart" element ={<Cart/>}/>
+    <Route path ="/login" element ={<Login/>}/>
+    <Route path ="/signup" element ={<Signup/>}/>
+
+
+  </Routes>
```
Comment (rahulcs754) [inline agrox-app/src/App.js:16]: you can set these all router one file and call single files for router

**Review by rahulcs754 — APPROVED** [review]
All Are initial setups that look good

File: agrox-app/src/App.js (hunk 1 above)
Comment (shubhamkumar648) [reply agrox-app/src/App.js:16]: Thank you I will do this

Comment (shubhamkumar648) [reply agrox-app/src/App.js]: done

**Pull request merged**